resources/fetching/fetchers/gcp/monitoring_fetcher.go

```diff
@@ -84,7 +84,7 @@ func (f *GcpMonitoringFetcher) Stop() {
 }
 
 func (g *GcpMonitoringAsset) GetMetadata() (fetching.ResourceMetadata, error) {
-	id := fmt.Sprintf("%s-%s", g.subType, g.Asset.ProjectId)
+	id := fmt.Sprintf("%s-%s", g.subType, g.Asset.ParentId)
```

Comment:
We must use a project id here.
In case we'll use an org id we'll get a resource id collision upon assessing multiple projects.